README.md

```diff
@@ -131,434 +131,436 @@ Table of Contents
 
 ## Source Code Repos
 
-  * [Azure DevOps](https://azure.microsoft.com/services/devops/) — Unlimited private repos (Git and TFS) for up to 5 users/team
```

Comment:
why remove all these?

Reply:
I think this is a desire to make a single format

Reply:
Judging by the size of the commit it was already pretty uniform :P and adding the / means fewer redirects when visiting the sites.

Reply:
> why remove all these?

Removing the "/"  was just a proposal to better uniform links, feel free to ignore it if you prefer

Reply:
You'd need to remove it from the PR, I cant selectively ignore that